content/en/docs/setup/independent/high-availability.md

```diff
@@ -467,7 +468,7 @@ Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**).
    networking:
      podSubnet: <podCIDR>
    apiServerCertSANs:
-   - <load-balancer-ip>
+   - <virtual-ip>
```

Comment:
`<private-ip>` as well as a host-specific fqdn should also be added here, otherwise it will complicate troubleshooting if VIP/LB is not available.

Reply:
On a related tangent, it might be a good idea to see if we can override the local kubelet, controller-manager, and scheduler configs to point to the local apiserver as well, having the locally deployed components reliant on the load balancer could cause issues at scale as well as introduce additional troubleshooting complexity around those components.

Reply:
Concerning the private-ip I agree, no problem.

Concerning the second comment -  I don't know how this could be accomplished. This PR is pretty much about using the VIP rather than the respective hosts address for the API Server. This is motivated by the following consideration: if a local API server instance crashes or shuts down while e.g. controller-manager is still up, we will have a potentially malfunctioning cluster.
Support for configuring more than one API server needs to be in kubelet, controller-manager and scheduler. I am not aware that it is; if it is indeed, that would of course be worthwhile to try out.

Reply:
I propose to first fix the current problem we have on the site (see my reply to @Bradamant3), then merge my PR with the change according to your first comment, then see whether we can accommodate your second comment (I honestly don't know whether it is possible). Agreed?